mo_gym/evaluation.py

```diff
@@ -13,43 +13,94 @@ def hypervolume(ref_point: np.ndarray, points: List[np.ndarray]) -> float:
 
     Returns:
         float: Hypervolume metric
-    """    
+    """
     return HV(ref_point=ref_point * - 1)(np.array(points) * - 1)
 
 
-def eval_mo(agent, env, w: np.ndarray, render: bool = False) -> Tuple[float, float, np.ndarray, np.ndarray]:
+def eval_mo(agent, env, scalarization, w: Optional[np.ndarray] = None,  render: bool = False) -> Tuple[float, float, np.ndarray, np.ndarray]:
```

Comment:
maybe let scalarization default to linear? so existing code which assumes linear (which will be most cases) works without passing this parameter to the function